nitro-protocol/test/contracts/ForceMove/challenge.test.ts

```diff
@@ -44,7 +45,7 @@ const chainId = process.env.CHAIN_NETWORK_ID;
 const participants = ['', '', ''];
 const wallets = new Array(3);
 const challengeDuration = 86400; // 1 day
-const outcome: Outcome = [{allocations: [], asset: Wallet.createRandom().address, metadata: '0x'}];
+const outcome: Outcome = [{allocations: [{destination: "0x00000000000000000000000096f7123e3a80c9813ef50213aded0e4511cb820f", amount: "0x01", allocationType: 1, metadata: "0x"}], asset: Wallet.createRandom().address, metadata: '0x'}];
```

Comment:
Was this change necessary?

Reply:
It was done for testing purposes. Removed in [`0fea82b`](https://github.com/statechannels/go-nitro/pull/508/commits/0fea82b1634e81ba892dad33ad304ae4d36db885)